aml-development/ozp-backend
# Orgsteward notification

AMLNG-745
(Part of https://github.com/aml-development/ozp-center/pull/893)

**Description**     
A process to notify and instruct content stewards to review and update, as needed, current listing information for all listing in their organization.

**Acceptance Criteria**   
- Only admins can send notifications to content stewards
- All content stewards receive the notification to update their apps
- Only content stewards receive the "update app" notification type
- Content stewards can click on a link in the in-site notification to go to the same Listing Management Recent Activity tab. (I chose this tab because all org stewards will have it and they can easily go to published listings from overview list on the left. They can also see which ones have been updated recently)
- The system notification type still functions as before there was no "Type" dropdown 

**How to test code**    
As bigbrother, 

1. Go to Center Settings (from the dropdown menu)
2. Go to the Notifications Tab
3. Select the Update Request type
4. See that the Notification text is populated with a default message (you can change this message if you like)
5. Set expires on date
6. Press send

Login as david, julia, obrien and wsmith to make sure that they have received the message and that it has the link as detailed in acceptance criteria

bigbrother should not receive the notification
hodor should not receive the notification
jones should not receive the notification

**Please Review**     
@aml-development/ozp-backend-team    

**Checklist**
